e2e_test/streaming/temporal_filter.slt

```diff
@@ -0,0 +1,25 @@
+statement ok
+SET RW_IMPLICIT_FLUSH TO true;
+
+statement ok
+create table t1 (v1 timestamp);
+
+# This statement should be correct for the next ~1000 years
```

Comment:
I love this idea! ☺️ Furthermore, can we add more test cases into this `slt` file? Only simple inserts are tested now. I think it's better to add tests for updates/deletes, or add some cases with other varients of predicate (e.g. `BETWEEN`)

Reply:
Indeed, we should do so.